.github/CONTRIBUTING.md

```
@@ -13,23 +13,60 @@ Side refers to the Sipe Design System, our primary UI component library aimed at
+echo 'eval "$(mise activate zsh)"' >> ~/.zshrc
+source ~/.zshrc
+```
+
+bash:
+```bash
+echo 'eval "$(mise activate bash)"' >> ~/.bashrc
+source ~/.bashrc
+```
+
+PowerShell:
+```powershell
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)